### PR TITLE
Initial agency docs

### PR DESCRIPTION
**Pull request by alex-s2**

### CODE REVIEW

File: docs/gameplay-component.md [hunk 1]
```diff
@@ -12,21 +12,68 @@
...
+Genie is a modular framework which provides a set of reusable components (known as "screens") which are common across all BBC games, as well as various core "engines", which handle GEL buttons, layout and accessibility.
+
+The current screens are:
+
+Title
+How to Play
+Pause
+Loading
+Select
+Results
```
Comment (SeanHolden) [inline docs/gameplay-component.md]: These need bullet points in front of them as in MarkDown they will be positioned inline.

**Comment by SeanHolden** [comment]
Looks fine to me so far.

File: docs/theming.md [hunk 2]
```diff
@@ -11,6 +11,8 @@
 
 ## What is in and out of scope for theming?
 
+Audio, Animations, Sprites/Graphics are in scope for theming. Alterations to the code other than configuration changes to support new assets are currently out of scope.
```
Comment (Thomas-James-Rose) [inline docs/theming.md]: Not sure if we have officially decided yet but will fonts be in scope for theming or are we just going to force people to use Reith?

File: docs/gameplay-component.md [hunk 3]
```diff
@@ -12,21 +12,68 @@
...
+All components extend the Genie "Screen" class. The `Screen` class extends `Phaser.State`, providing the `Context` to objects that extend from it. Once your game component extends Screen and has functionality, import your component into main as you would any other screen, and add it into the sequence at the desired point.
+
 
 ## How do I test my component in the Genie framework?
 
+You can preview your game without bundling it through Webpack by running it in a live server using `npm start`, and viewing it in a browser at http://localhost:8080/.
+
+The qaMode query string may be added to the end to view the game in QA Mode. This gives additional console output, and if you press "q", you can see the layout overlay. http://localhost:8000/?qaMode=true.
+
+To build your game using Webpack, use `npm run build`.
```
Comment (Leonie-) [inline docs/gameplay-component.md]: Is it worth mentioning about easily switching  a game's theme by using the query string? e.g.`http://localhost:8080/?theme=<themeName>`?

**Comment by Leonie-** [comment]
One small comment, otherwise looks good :icecream:

**Pull request merged**